README.md

```diff
@@ -59,12 +59,15 @@ Due to the large amount of records anticipated to be created, you must create an
 return [
 
     'morphs' => [
-        0 => App\Models\User::class
+        1 => App\Models\User::class
     ],
 
 ];
 ```
 
+> [!CAUTION]
+> `0` is not a valid key for the morph map, as it will get resolved to `false` in Laravel's morphing logic.
```

Comment:
Just to clarify on this for anyone interested: It's specifically the `buildDictionary` logic that is used when eager loading.

```php
    protected function buildDictionary(Collection $models)
    {
        foreach ($models as $model) {
            if ($model->{$this->morphType}) {
                $morphTypeKey = $this->getDictionaryKey($model->{$this->morphType});
                $foreignKeyKey = $this->getDictionaryKey($model->{$this->foreignKey});

                $this->dictionary[$morphTypeKey][$foreignKeyKey][] = $model;
            }
        }
    }
```

The `if ($model->{$this->morphType}) {` check is what get resolved to false when the morph map value is a 0.  This prevents the model from being added to dictionary and ultimately results in attempts at eager loading relationships such as `entity` always returning null when the relationship is to the model with a morph map value of 0.